src/Files.App/Views/LayoutModes/DetailsLayoutBrowser.xaml

```diff
@@ -974,7 +975,7 @@
 									<!--  Item Name  -->
 									<Grid
 										Width="{Binding ColumnsViewModel.NameColumn.LengthIncludingGridSplitter.Value, ElementName=PageRoot, Mode=OneWay}"
-										Padding="12,0,0,0"
+										Padding="10,0,0,0"
```

Comment:
This should be 12, otherwise it doesn't line up with the header. That said, let's move the alignment changes into another PR so that we can focus on the divider brushes.

Reply:
Fixed